app/src/bcsc-theme/features/verify/VerifyIdentityStack.tsx

```diff
@@ -47,15 +47,28 @@ import VideoInstructionsScreen from './send-video/VideoInstructionsScreen'
 import VideoReviewScreen from './send-video/VideoReviewScreen'
 import VideoTooLongScreen from './send-video/VideoTooLongScreen'
 import NicknameAccountScreen from '../account/NicknameAccountScreen'
+import AccountSelectorScreen from '../account/AccountSelectorScreen'
+import { BCState } from '@/store'
 
 const VerifyIdentityStack = () => {
   const Stack = createStackNavigator<BCSCVerifyIdentityStackParams>()
   const theme = useTheme()
   const { t } = useTranslation()
   const defaultStackOptions = useDefaultStackOptions(theme)
+  const [store] = useStore<BCState>()
 
   return (
     <Stack.Navigator screenOptions={{ ...defaultStackOptions, headerShown: true, title: '' }}>
+      {store.bcsc.nicknames.length > 0 && (
+        <Stack.Screen
+          name={BCSCScreens.AccountSelector}
+          component={AccountSelectorScreen}
+          options={{
+            headerShown: true,
+            title: t('Unified.AccountSelector.Header'),
+          }}
+        />
+      )}
```

Comment:
question: Are we supposed to see this screen as the first step VerifyIdentityStack (setup steps)?

Or do we want to see this screen as the first screen in the main stack?

Or both...

@bryce-mcmath